dspace-server-webapp/src/main/java/org/dspace/app/rest/BundleUploadBitstreamController.java

```diff
@@ -87,7 +87,7 @@
     @RequestMapping(method = RequestMethod.POST, headers = "content-type=multipart/form-data")
     @PreAuthorize("hasPermission(#uuid, 'BUNDLE', 'ADD') && hasPermission(#uuid, 'BUNDLE', 'WRITE')")
     public ResponseEntity<ResourceSupport> uploadBitstream(HttpServletRequest request, @PathVariable UUID uuid,
-                                           @RequestParam("file") MultipartFile uploadfile,
+                                           @RequestParam(value = "file", required = false) MultipartFile uploadfile,
```

Comment:
why we need to set it as not required if we check for not null later?

Reply:
Because the linked Rest Contract required the 422 exception.
Currently REST just can't find a matching method if there's no file